Charter.md

```diff
@@ -0,0 +1,135 @@
+# PlanetarPy Charter
+
+PlanetaryPy is governed by the members of the Technical Committee (TC).
+
+## Statement of Purpose
+
+PlanetaryPy is a community effort to develop a core package
+for planetary science in Python and foster interoperability between
+Python planetary science packages.
+
+## Responsibilities
+
+The PlanetaryPy TC responsibilities include:
+
+* Technical direction
+* Setting release dates.
+* Release quality standards.
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+
+## Membership
+
+### Technical Committee Members
+
+For the current list of TC members, see the [member list](Members.md).
+
+
+TC seats are not time-limited.  There is no fixed size of the 
+TC.  However, the expected target is between 3 and 12, to ensure adequate
+coverage of important areas of expertise, balanced with the ability
+to make decisions efficiently.
+
+There is no specific set of requirements or qualifications for 
+TC membership beyond these rules.
+
+The TC may add additional members by consensus. If there
+are any objections to adding any individual member, an attempt
+should be made to resolve those objections following the 
+Consensus Seeking Process.
+
+A TC member may be removed from by voluntary resignation, or by
+consensus of all other members.
+
+Changes to TC membership should be posted in the agenda, and may be
+suggested as any other agenda item (see "Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full
+TC is not in attendance to participate, then the addition or removal
+is added to the agenda for the subsequent meeting.  This is to ensure
+that all members are given the opportunity to participate in all
+membership decisions.  If a TC member is unable to attend a meeting
+where a planned membership decision is being made, then their consent
+is assumed.
+
```

Comment:
Do we have/need a general comment somewhere about lazy consent?

Reply:
The very last item in the document is about the Consensus model.